server/src/main/resources/db/migration/updated natural textactions examples.sql

```diff
@@ -0,0 +1,192 @@
+UPDATE natural_text_action_examples
```

Comment:
Please change the migration filename. Have the same format as other files in the migration folder

Reply:
Will update.